Tests/LanguageServerProtocolTests/LanguageServerProtocolTests.swift

```diff
@@ -14,6 +14,11 @@ import LanguageServerProtocol
 
 import XCTest
 
+fileprivate func AssertDataIsString(_ data: Data, expected: String) {
+  let got = String(decoding: data, as: UTF8.self)
+  XCTAssertEqual(got, expected)
```

Comment:
Should add `file: StaticString = #file, line: UInt = #line` and forward these to `XCTAssertEqual` so that failures are not all attributed to the helper method.

Reply:
Done